src/modules/upcoming-events.module/module.html

```diff
@@ -3,5 +3,6 @@
 
 <div class="upcoming-events">
     <h2>Upcoming Events</h2>
-    <div id="upcoming-events__module" data-portal-id="{{ portal_id }}"></div>
+    {% set appSlug = content_by_id(module.event_page).slug %}
+    <div id="upcoming-events__module" data-portal-id="{{ portal_id }}" data-slug="{{ appSlug }}"></div>
```

Comment:
For this module, I think we probably don't want to use an `id` and may want to consider how we can support multiple instances on the same page.